src/Microsoft.SqlTools.ServiceLayer/DacFx/Contracts/DeploymentOptions.cs

```diff
@@ -436,9 +440,22 @@ public DeploymentOptions(DacDeployOptions options)
             // Setting Display names for all dacDeploy options
             SetDisplayNameForOption();
 
+            // Preparing Include Object types options table
+            CreateIncludeObjectsTable();
+
             SetOptions(options);
         }
 
+        /// <summary>
+        /// Sets include objects enum values and number in to the dictionary
```

Comment:
what are the numbers of these include objects?

Reply:
Actually its enum names and enum values, ill update the comment, Thanks